algorithms/calculator/math_parser.py

```diff
@@ -91,6 +91,8 @@ def parse(expression):
                 current = ""
             if i in __operators__ or i in __parenthesis__:
                 result.append(i)
+            elif i == ' ':
+                pass

```

Comment:
Can improve this by
```python
elif not i.isspace():
    raise ValueError("invalid syntax " + i)
```

Reply:
@goswami-rahul hey! good to have you back :))